config/_default/menus/menus.en.toml

```diff
@@ -56,18 +56,18 @@
     weight = 400
 [[footer]]
     parent = "column1"
-    name = "Documentation"
-    url = "/docs/"
+    name = "Download"
+    url = "/docs/install/"
     weight = 410
 [[footer]]
     parent = "column1"
-    name = "Language Guide"
-    url = "/docs/language-guide/"
+    name = "Get started"
+    url = "/docs/get-started/"
```

Comment:
I've updated the alpha site map to reflect this sub page off of docs.

Reply:
We will omit /docs/get-started/ as a page, and instead use that as a header in /docs